modelbaker/iliwrapper/ilicache.py

```diff
@@ -879,6 +895,12 @@ def on_download_status(self, dataset_id):
         if len(self.downloaded_files) == len(self.file_ids) == self.model.rowCount():
             self.download_finished.emit()
 
+    def clear_cache():
+        if not QDir().exists(IliToppingFileCache.CACHE_PATH):
```

Comment:
Can't we just have the function in the top class `IliCache` and with `self` the correct variable is used?

Reply:
I wanted to use static functions to avoid instantiating the classes just to clear the cache.
But I didn't see that they are all inheriting from IliCache. I'll change that